garage/np/algos/batch_polopt.py

```diff
@@ -115,6 +118,155 @@ def log_diagnostics(self, paths):
         self.policy.log_diagnostics(paths)
         self.baseline.log_diagnostics(paths)
 
+    def process_samples(self, itr, paths):
+        baselines = []
+        returns = []
+
+        if hasattr(self.baseline, 'predict_n'):
```

Comment:
hmm it is probably more appropriate to calculate the baseline in the training loop. @CatherineSue what do you think?

Reply:
Sorry just saw this. I agree.

Reply:
i think we should file an issue for this one rather than asking @zequnyu to fix it in this PR

Reply:
@zequnyu did you file an issue for this?

Reply:
Filed